src/go/privileged-service/pkg/manage/uninstall.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright © 2021 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manage
+
+import (
+	"fmt"
+
+	"golang.org/x/sys/windows/svc/eventlog"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+// Uninstall Service removes the Rancher Desktop Privileged Service process from Windows Services
+func UninstallService(name string) error {
+	m, err := mgr.Connect()
+	if err != nil {
+		return fmt.Errorf("connect to service [%s] failed: %w", name, err)
+	}
+	defer m.Disconnect()
+	s, err := m.OpenService(name)
+	if err != nil {
+		return fmt.Errorf("service [%s] is not installed: %w", name, err)
+	}
+	defer s.Close()
+	if err = s.Delete(); err != nil {
```

Comment:
Do we need to try to stop the service first, if it's running? I assume this will be used in the uninstaller, in which case we should make sure it's stopped so that the file can be deleted.